app/assets/stylesheets/tags.scss

```diff
@@ -0,0 +1,27 @@
+.tag-list {
+  margin-top: 2rem;
+}
+
+.tags {
+  margin-bottom: 0.8rem;
+}
+
+.tag {
+  @extend .chip;
+  background-color: #E6E18F;
+  color: #92977E;
+}
+
+.tag-individual {
+  margin-bottom: 1rem;
```

Comment:
Properties should be ordered border-radius, font-size, height, line-height, margin-bottom, padding, vertical-align